client/app/pod/popular.controller.js

```diff
@@ -1,6 +1,6 @@
 angular.module('throwcast.popular')
 
-.controller('PopularController', function ($scope, $http) {
+.controller('PopularController', function ($scope, $http, userService) {
```

Comment:
You aren't using the `userService` right now. Remove this injection until it is needed.